evalml/tests/data_checks_tests/test_invalid_targets_data_check.py

```diff
@@ -70,6 +70,19 @@ def test_invalid_target_data_check_numeric_binary_classification_error():
     }
 
 
+def test_invalid_target_data_check_multiclass_two_examples_per_class():
+    X = pd.DataFrame()
+    invalid_targets_check = InvalidTargetDataCheck("multiclass")
+    assert invalid_targets_check.validate(X, y=pd.Series([0, 1, 1, 2, 2])) == {
+        "warnings": [],
+        "errors": [DataCheckError(message="Target does not have two examples per class which is required for multi-classification",
+                                  data_check_name=invalid_targets_data_check_name,
+                                  message_code=DataCheckMessageCode.TARGET_BINARY_NOT_TWO_EXAMPLES_PER_CLASS,
+                                  details={"least_populated_class_count": 1,
```

Comment:
RE previous comment, I think it'd be helpful to add a test where `y=pd.Series([0, 1, 1, 2, 2, 3])` and then return in the details information about 0 or 3.

Reply:
Added